watch-and-show-snapshots.js

```diff
@@ -0,0 +1,28 @@
+let fs = require('fs')
+let chokidar = require('chokidar')
+let parseGitignore = require('parse-gitignore')
+
+let updateAndShowSnaphots = require('./update-and-show-snapshots')
+
+async function watchAndShowSnaphots (print, error, cwd, filter) {
+  let ignored = ['.git', 'node_modules']
+
+  if (fs.existsSync(`${ cwd }/.gitignore`)) {
+    ignored = [...new Set([
+      ...ignored,
+      ...parseGitignore(fs.readFileSync(`${ cwd }/.gitignore`)).map(
```

Comment:
We can believe that `cwd` will be the root of the project.

The only way to find project root is to find `package.json`.